QuizApp.Services/TestService.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using QuizApp.DAL.Repository.Contracts;
+using QuizApp.DTO;
+using QuizApp.Services.Contracts;
+
+namespace QuizApp.Services
+{
+	public class TestService : ITestService
+	{
+		private readonly IRepositoryWrapper _repository;
+		private readonly IMapper _mapper;
+		public TestService(IRepositoryWrapper repository, IMapper mapper)
+		{
+			_repository = repository;
+			_mapper = mapper;
+		}
+
+		public async Task<List<TestDto>> GetAll()
+		{
+			var tests = await _repository.Test.GetAll()
+				.Include(t => t.TestQuestions)
```

Comment:
I would move this code to the repository as it's a part of the logic of just loading data.

Reply:
Moved Include().ThenInclude() to repository but ToListAsync() is still in the service
Is it ok, or should it also be moved to repository?

Reply:
`ToListAsync()` should also be in the repository